django_project/frontend/utils/metrics.py

```diff
@@ -0,0 +1,37 @@
+from collections import Counter
+
+from django.db.models import Sum
+from property.models import Property
+
+
+def calculate_population_categories(queryset):
+    POPULATION_CATEGORY_DICT = {
```

Comment:
In Python convention, uppercase names are used for constants, which don't change once assigned.